go.mod

```diff
@@ -3,8 +3,11 @@ module github.com/PhilLar/Images-back
 go 1.12
 
 require (
+	github.com/golang-migrate/migrate v3.5.4+incompatible
```

Comment:
Please run `go mod tidy` to remove unused dependencies from `go.mod` file

Reply:
About `go mod tidy`: https://github.com/golang/go/wiki/Modules#how-to-prepare-for-a-release